src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1214,12 +1214,12 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
-		// Updates if we should show Description
-		$ticket->show_description = isset( $ticket->show_description ) && tribe_is_truthy( $ticket->show_description ) ? 'yes' : 'no';
-
 		/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
 		$tickets_handler = tribe( 'tickets.handler' );
 
+		// Updates if we should show Description
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 066144d01f0cb0fe9e17a6c01cf633da</sub>

Reply:
```suggestion
		// Updates if we should show Description.
```